tests/README.md

```diff
@@ -0,0 +1,126 @@
+Here is a comprehensive but evolutive list of the unit and integration tests to perform.
+
+
+airdrops.cairo :
```

Comment:
You can take inspiration here from https://github.com/starknetfndn/defispring , the contract has tests done